corgi/collectors/brew.py

```diff
@@ -486,23 +485,23 @@ def get_container_build_data(self, build_id: int, build_info: dict[str, Any]) ->
 
         go_stdlib_version = ""
         remote_sources: dict[str, tuple[str, str]] = {}
+        is_rhcos_build = build_info.get("cg_name") == "coreos-assembler"
```

Comment:
you might consider defining a class variable (RHCOS="coreos-assembler") at the beginning of brew class and refer to that here - to give hints to future visitors ...

Reply:
done

Reply:
done